### ISSUE TITLE
Anyone can get the result comparable to the model released?

### ISSUE THREAD

**Issue by Edwardmark** [Goingqs/PyramidBox#28]

**Comment by Edwardmark** [comment]
I train from scratch, and get result of WIDERFACE(92.5,91.3,85.2) on the validation set.

**Comment by YueXiNPU** [comment]
@Edwardmark @Goingqs , I train from scratch, but get result of widerface (48.6%, 55.6%, 54.0%) on val. Can you share the training setting?
